tests/object_file_system_test.php

```diff
@@ -177,6 +177,45 @@ public function test_copy_object_from_local_to_external_by_hash_if_not_local_and
         $this->assertEquals(OBJECT_LOCATION_ERROR, $location);
     }
 
+     public function test_create_external_orphaned_object_record_in_the_table() {
+        global $CFG, $DB;
+
+        $this->filesystem = new test_file_system();
+        $file = $this->create_duplicated_file(time());
+        $filehash = $file->get_contenthash();
+        $objectrecord = $DB->get_record('tool_objectfs_objects',array('contenthash' => $filehash));
+
+        manager::update_object($objectrecord, OBJECT_LOCATION_ORPHANED);
+
+        $this->assertTrue($this->is_externally_readable_by_hash($filehash));
+    }
+
+    public function test_if_external_orphaned_object_is_deleted_after_delaydeleteexternalobject_setting() {
```

Comment:
This method is not currently testing something. I suggest we delete it. If you want to test the task execution, I suggest you first create orphaned files with an expired `orphanedtime` and then run the `execute` function from the task.

Reply:
But I think we can get a pass on running tests for the task. They are currently not testing their tasks.